src/_includes/layouts/base.njk

```diff
@@ -26,6 +26,8 @@
     <title>FlowForge &#x2022; DevOps for Node-RED</title>
     {% endif %}
 
+    <meta rel="canonical" href="{{ page.url }}">
```

Comment:
PR title says "all blog posts", but this will impact all pages on the website, which is the intension?

Reply:
@Pezmc I'm not aware of tricks to do so just for blogs. I don't think it matters either, it's not a bad practise this way.

But there's a few duplicate pages, only as blogs, which is why it's specifically about blogs.